js/components/Editor.js

```diff
@@ -50,36 +53,73 @@ const Editor = React.createClass({
   },
 
   onGutterClick(cm, line, gutter, ev) {
-    let info = cm.lineInfo(line);
-    cm.setGutterMarker(
-      line,
-      "breakpoints",
-      info.gutterMarkers ? null : makeMarker()
-    );
-
```

Comment:
As said in my other comment, I'd prefer if you reverted all the changes in this file. I'm actively working on it already.